src/Protocol/Protocol.h

```diff
@@ -259,9 +256,6 @@ class cProtocol
 	/** Returns the protocol-specific packet ID given the protocol-agnostic packet enum. */
 	virtual UInt32 GetPacketID(ePacketType a_Packet) = 0;
 
-	/** Converts eMonsterType to protocol-specific mob types */
-	virtual UInt32 GetProtocolMobType(eMonsterType a_MobType) = 0;
```

Comment:
Why was this moved?

Reply:
It was too high up and forced ProtocolRecognizser to have a dummy `return 0` implementation, now the root of the inheritance chain starts lower, from a concrete protocol 1.8.